bots/bridge-tester/test/_initialization/initialize.test.ts

```diff
@@ -79,49 +77,60 @@ describe.skip("Initialize parachain state", () => {
-      "StableParachainConfirmations",
-      new BN(stableParachainConfirmationsToSet),
-      alice
-    );
-    const stableBitcoinConfirmations =
-      await interBtcAPI.btcRelay.getStableBitcoinConfirmations();
-    assert.equal(
-      stableBitcoinConfirmationsToSet,
-      stableBitcoinConfirmations,
-      "Setting the Bitcoin confirmations failed"
-    );
-    const stableParachainConfirmations =
-      await interBtcAPI.btcRelay.getStableParachainConfirmations();
-    assert.equal(
-      stableParachainConfirmationsToSet,
-      stableParachainConfirmations,
-      "Setting the Parachain confirmations failed"
-    );
-    await bitcoinCoreClient.mineBlocks(3);
-  });
+    it("should set the stable confirmations and ready the BTC-Relay", async () => {
+        // Speed up the process by only requiring 0 parachain and 0 bitcoin confirmations
+        const stableBitcoinConfirmationsToSet = 0;
+        const stableParachainConfirmationsToSet = 0;
+        let [stableBitcoinConfirmations, stableParachainConfirmations] = await Promise.all([
+            userInterBtcApi.btcRelay.getStableBitcoinConfirmations(),
+            userInterBtcApi.btcRelay.getStableParachainConfirmations(),
+        ]);
+
+        if (stableBitcoinConfirmations != 0 || stableParachainConfirmations != 0) {
+            console.log("Initializing stable block confirmations...");
+            await setRawStorage(
+                api,
+                api.query.btcRelay.stableBitcoinConfirmations.key(),
+                api.createType("u32", stableBitcoinConfirmationsToSet),
+                sudoAccount
+            );
+            await setRawStorage(
+                api,
+                api.query.btcRelay.stableParachainConfirmations.key(),
+                api.createType("u32", stableParachainConfirmationsToSet),
+                sudoAccount
+            );
+            await bitcoinCoreClient.mineBlocks(3);
+            [stableBitcoinConfirmations, stableParachainConfirmations] = await Promise.all([
+                userInterBtcApi.btcRelay.getStableBitcoinConfirmations(),
+                userInterBtcApi.btcRelay.getStableParachainConfirmations(),
+            ]);
+        }
+        assert.equal(
+            stableBitcoinConfirmationsToSet,
+            stableBitcoinConfirmations,
+            "Setting the Bitcoin confirmations failed"
+        );
+        assert.equal(
+            stableParachainConfirmationsToSet,
+            stableParachainConfirmations,
+            "Setting the Parachain confirmations failed"
+        );
+    });
+
 
   it("should issue 0.1 InterBTC", async () => {
     const wrappedToIssue = newMonetaryAmount(0.00007, wrappedCurrency, true);
     await issueSingle(
-      interBtcAPI,
+      userInterBtcApi,
       bitcoinCoreClient,
-      alice,
+      user1Account,
       wrappedToIssue,
       vault_id
     );
   });
 });
+function setRawStorage(api: ApiPromise, arg1: string, arg2: u32, sudoAccount: KeyringPair) {
```

Comment:
Any particular reason behind keeping the unimplemented method around?

Reply:
I'll keep it in for now, it's meant to just get the `bridge-tester` build to pass since this bot is not currently used